PSReadLine/KillYank.cs

```diff
@@ -70,7 +70,7 @@ private void Kill(int start, int length, bool prepend)
                 _buffer.Remove(start, length);
                 _current = start;
                 Render();
-                if (_killCommandCount > 0)
+                if (_killCommandCount > 0 && _killIndex >= 0)
```

Comment:
Do you think it would be cleaner to not increment `_killCommandCount` when `length == 0`? In other words, move [this line](https://github.com/lzybkr/PSReadLine/blob/e1dcaaededb4e672491dd4b782a2b213b5e97c1d/PSReadLine/KillYank.cs#L102) up, or maybe just add an early return when `length == 0`, that would make the code less indented.